packages/sit-onyx/src/components/OnyxRipple/OnyxRipple.vue

```diff
@@ -0,0 +1,86 @@
+<script lang="ts" setup>
+import { computed, ref } from "vue";
+import { useRipple, type RippleConfig } from "../../composables/useRipple";
+
+const rippleTrigger = ref<HTMLElement>();
+
+const config = computed<RippleConfig>(() => ({
+  color: "var(--onyx-ripple-color, var(--onyx-color-base-primary-600))",
+  container: rippleTrigger,
+}));
+
+const { ripples, hideRipple, events } = useRipple(config);
+
+defineExpose({
+  events,
+});
+</script>
+
+<template>
+  <span ref="rippleTrigger" class="onyx-ripple">
+    <transition-group name="onyx-ripple" @after-enter="hideRipple($event as HTMLElement)">
+      <span
+        v-for="[key, r] in ripples"
+        :key="key"
+        class="onyx-ripple__element"
+        :style="{
+          left: r.left,
+          top: r.top,
+          width: r.radius,
+          height: r.radius,
+          backgroundColor: r.backgroundColor,
+        }"
+        :data-rippleid="key"
+      ></span>
+    </transition-group>
+  </span>
+</template>
+
+<style lang="scss">
+@use "../../styles/mixins/layers.scss";
+@use "../../styles/mixins/density.scss";
```

Comment:
The density import is unused :)

Also can you wrap the styles in `@include layers.component() {}` as described [here](https://onyx.schwarz/principles/contributing.html#developing-components)? :)